### DISCUSSION TITLE
My first Sozi video: Advertisement for Code & Supply's Compensation Survey

### DISCUSSION THREAD

**Discussion by colindean** [senshu/Sozi#640] [Show and tell]
https://www.youtube.com/watch?v=jcc_AEjoF1Y

My time to produce this was pretty limited, so I kept the base SVG _very_ basic and the Sozi orchestration uncomplicated. I found the tool easy to use and its video output to be very high quality.

My greatest want in the process is some indication of the total length of the animation and a more explicit way to see all of the timings of the presentation together. It was a bit challenging to tweak slide timings to reduce the runtime of the whole animation to under 30 seconds.

A feature request would be to provide a way to do voiceovers. I think an MVP might be providing an audio file for each slide and marking if the audio should play at the transition's start or end. Sozi would then by default set the timeout for the slide to be the length of the audio file, but the user could change this. My approach given the short length of the animation above was to ask the person recording the voiceover to simply record all of the audio in one or two takes.

Thank you for making a great tool. I like working with Inkscape a lot, and its usefulness combined Sozi just definitely earned Sozi a spot in my marketing and presentation toolkit.

**Comment by aumouvantsillage** [comment]
Thanks for sharing. It is simple but effective.

I agree that adding audio can be challenging. Sozi has been designed as a live presentation tool. Video export is provided for convenience but it is not a central part of the application. Users who choose Sozi with the intent of creating videos will probably find that it lacks many features.
